tests/python/PyImportSortBearTest.py

```diff
@@ -21,3 +21,12 @@
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
               'known_first_party_imports': 'abc'})
+
+PyImportSortBearIgnoredConfigsTest = verify_local_bear(
+    PyImportSortBear,
+    ('import sys\n\nimport datetime\n',
+     'from sys import *\n\nimport datetime\n'),
+    ('import sys\nimport datetime\n',
```

Comment:
Please only keep the second bad file. The first one is also bad because of the alphabetical order and does not add anything.